### PR TITLE
 Removing JavaScript

### PR DESCRIPTION
**Pull request by KarenDouglas**
#40 
- removes javaScript

### CODE REVIEW

File: app/root.tsx [hunk 1]
```diff
@@ -42,6 +43,13 @@ export const meta: MetaFunction = () => ({
 });
 
 export default function App() {
+
+  const matches = useMatches();
+
+  const includeScripts = matches.some(
+    (match) => match.handle?.hydrate
+  );
```
Comment (brookslybrand) [inline app/root.tsx:51]: Ah nice, where'd you find this trick?

Comment (KarenDouglas) [reply app/root.tsx:51]: I found it in the [remix docs](https://remix.run/docs/en/v1/guides/disabling-javascript).

Comment (KarenDouglas) [reply app/root.tsx:51]: :)

**Review by grantnorwood — APPROVED** [review]
Looks good!  And it saved 8 JS requests, somewhere near 0.2mb, and it loads faster 🙂

image

⬇️ 

image

**Comment by grantnorwood** [comment]
I dunno if we got to 💯 across the board by removing JS, I didn't measure before I merged.  But we're there now!

image

**Pull request merged**